themes/bootstrap3/templates/search/advanced/layout.phtml

```diff
@@ -196,7 +196,8 @@
         <?php endforeach; ?>
       <?php endif; ?>
       <h2><?=$this->transEsc("Search Tips")?></h2>
-      <div class="facet-group">
+      <?php // Data-title attribute is for analytics use.  Do not remove. ?>
+      <div class="facet-group" data-title="<?=$this->escapeHtmlAttr("Search Tips") ?>">
```

Comment:
@maccabeelevine, this is not a real facet group -- it's just a static list of help links that happens to recycle the styling of a facet group. I don't think we want to treat as if it were a facet for analytics purposes. @crhallberg, should we re-style this so it's less misleading?

Reply:
So yes dealing with the Search Tips sidebar here is a hack, all I can say is that it's a pre-existing hack (using facet-group) that I took advantage of.  I am equally interested in capturing clicks on those advanced search help links as I am the real facets. (Lehigh just customized that help text, and we are interested in what use it gets.)  I did the two together in one PR since they were already implemented the same way, and it made the analytics seamless.  If we have to separate it that's fine, but there should be some new sidebar heading / item standard with equivalent data-elements for each, and I'll track it separately in GTM.

Reply:
Thanks for the context, @maccabeelevine. Since it's intentional, I'll let it slide and merge this, though I definitely don't like how hacky the whole thing feels. As you say, when we fix the pre-existing issue, we can also address this one. And if we don't get to that right away, at least you'll have a workable approach in the meantime.